src/plots/piePlot.ts

```diff
@@ -157,6 +157,42 @@ export module Plots {
       return this;
     }
 
+    /**
+     * Returns the PlotEntity at the query point.
+     * 
+     * @param {Point} queryPoint
+     * @param {PlotEntity} The PlotEntity at the point, or null if no PlotEntity can be found. 
+     */
+    public entityAt(queryPoint: Point): PlotEntity {
```

Comment:
For consistency the method should be named `entitiesAt()`, even though we won't ever return more than one slice.